lib/maths/unittest/CTimeSeriesDecompositionTest.cc

```diff
@@ -113,6 +114,7 @@ const core_t::TTime HOUR = core::constants::HOUR;
 const core_t::TTime DAY = core::constants::DAY;
 const core_t::TTime WEEK = core::constants::WEEK;
 const core_t::TTime YEAR = core::constants::YEAR;
+const ml::maths::CModelAddSamplesParams::TModelChangeCallback NOOP;
```

Comment:
It would be good to test, in at least one test, that the annotations are actually created.

Reply:
How should I go about the unit testing in this case? Should I call `addPoint` method with a model annotation callback and verify it got called? 
Could you point me at the test case which I can take as a base?